<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Text App</string>
  </property>
  <property name="windowIcon">
   <iconset theme="insert-text">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="topLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelModeInfo">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>13</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Выберите режим записи:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxMode">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <bold>false</bold>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Первый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Второй</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Третьий</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Четвертый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Пятый</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayoutSize">
       <item>
        <widget class="QLabel" name="labelElementsInfo">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <pointsize>13</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Кол-во элементов:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>140</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxElements">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonExit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>700</y>
      <width>190</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>381</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutIO">
       <item>
        <widget class="QLabel" name="labelInputInfo">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Ввод:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditInput"/>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOutputInfo">
         <property name="font">
          <font>
           <family>Liberation Sans</family>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Вывод:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditOutput">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>381</width>
        <height>542</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutStruct">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="bottomMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="labelModelInfo">
             <property name="font">
              <font>
               <family>URW Gothic</family>
               <pointsize>15</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>4</number>
             </property>
             <property name="text">
              <string>Модель:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEditModel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777214</width>
               <height>100</height>
              </size>
             </property>
             <property name="lineWrapMode">
              <enum>QPlainTextEdit::WidgetWidth</enum>
             </property>
             <property name="plainText">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="labelMileage">
             <property name="font">
              <font>
               <family>URW Gothic</family>
               <pointsize>15</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>4</number>
             </property>
             <property name="text">
              <string>Пробег:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxMileageInfo">
             <property name="maximum">
              <number>999999</number>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QLabel" name="labelMaxSpeedInfo">
               <property name="font">
                <font>
                 <family>URW Gothic</family>
                 <pointsize>15</pointsize>
                 <bold>false</bold>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="lineWidth">
                <number>4</number>
               </property>
               <property name="text">
                <string>Максимальная скорость:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="doubleSpinBoxMaxSpeed">
               <property name="maximum">
                <double>9999.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_24">
         <item>
          <widget class="QLabel" name="labelTypeInfo">
           <property name="font">
            <font>
             <family>URW Gothic</family>
             <pointsize>15</pointsize>
             <bold>false</bold>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="lineWidth">
            <number>4</number>
           </property>
           <property name="text">
            <string>Тип:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="2">
            <widget class="QRadioButton" name="radioButtonTypeC">
             <property name="text">
              <string>C - Cruiser</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="radioButtonTypeS">
             <property name="text">
              <string>S - Sport</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QRadioButton" name="radioButtonTypeD">
             <property name="text">
              <string>D - Dirt</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="radioButtonTypeA">
             <property name="text">
              <string>A - Adventure</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QRadioButton" name="radioButtonTypeT">
             <property name="text">
              <string>T - Touring</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_15">
           <item>
            <widget class="Line" name="line_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelDamageInfo">
             <property name="font">
              <font>
               <family>URW Gothic</family>
               <pointsize>15</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>4</number>
             </property>
             <property name="text">
              <string>Повреждения:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxIsDamaged">
             <property name="text">
              <string>Есть повредждения</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonAddElement">
         <property name="font">
          <font>
           <family>URW Gothic</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить мотоцикл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_19">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <action name="actionNiggers">
   <property name="text">
    <string>Niggers</string>
   </property>
  </action>
  <action name="actionBiggers">
   <property name="text">
    <string>Biggers</string>
   </property>
  </action>
  <action name="actionDiggers">
   <property name="text">
    <string>Diggers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
